build-test.xml

```diff
@@ -8516,38 +8559,51 @@ ANT_OPTS=${env.ANT_OPTS}</echo>
 			<then>
 				<mkdir dir="${app.server.parent.dir}" />
 
-				<tstamp>
-					<format pattern="yyyyMMddkkmmssSSS" property="tstamp.value" />
-				</tstamp>
+				<local name="bundle.file.name" />
+
+				<basename file="${test.build.bundle.zip.url}" property="bundle.file.name" />
 
 				<mirrors-get
-					dest="${tstamp.value}.zip"
+					dest="${bundle.file.name}"
 					src="${test.build.bundle.zip.url}"
 				/>
 
-				<unzip
-					dest="${app.server.parent.dir}"
-					src="${tstamp.value}.zip"
+				<decompress
+					dest="${project.dir}"
+					src="${bundle.file.name}"
+				/>
+
+				<move
+					todir="${app.server.parent.dir}"
+					verbose="true"
 				>
-					<patternset
+					<fileset
+						dir="${project.dir}"
 						includes="liferay-*/**"
 					/>
 					<mapper
 						from="[^/]+/(.*)"
 						to="\1"
 						type="regexp"
 					/>
-				</unzip>
+				</move>
 
-				<delete file="${tstamp.value}.zip" />
+				<delete includeemptydirs="true">
```

Comment:
This deletion is kind of scary, isn't it? We'll have to remember that any files or dirs in the root dir of liferay-portal that start with "liferay-" are effectively designated as temporary. Maybe it would be better to extract everything to a temp subdir with a more unique name.

Reply:
I think you're right. I'll change that to point to a temp directory